src/components/common/TmBalance.vue

```diff
@@ -132,7 +132,10 @@
 
         <div class="table-cell actions">
           <div class="icon-button-container">
-            <button class="icon-button circle-send-button" @click="onSend()">
+            <button
```

Comment:
classical example that this need to be refactored into a `LiBalance` @jbibla

Reply:
sounds like a good plan 👍